publish/binaries/BUILD.bazel

```diff
@@ -46,15 +45,11 @@ BINARIES = {
     "ic-consensus-pool-util": "//rs/artifact_pool:ic-consensus-pool-util",
     "ic-crypto-csp": "//rs/crypto:ic-crypto-csp",
     "ic-nns-init": "//rs/nns/init:ic-nns-init",
-    "ic-prep": "//rs/prep:ic-prep",
     "ic-recovery": "//rs/recovery:ic-recovery",
     "ic-regedit": "//rs/registry/regedit:ic-regedit",
     "ic-registry-replicator": "//rs/orchestrator/registry_replicator:ic-registry-replicator",
     "ic-replay": "//rs/replay:ic-replay",
-    "ic-rosetta-api": "//rs/rosetta-api:ic-rosetta-api",
```

Comment:
@mbjorkqvist @NikolasHai , I believe you release the rosetta nodes using different build targets than this one but please confirm

Reply:
I also believe that this is the case, but @NikolasHai would be the best person to confirm.

Reply:
We release ICP rosetta through a docker container, like [this](https://sourcegraph.com/github.com/dfinity/ic/-/blob/rs/rosetta-api/BUILD.bazel?L216). We never make a release for the binary directly.

Reply:
ok, so I will proceed and merge the PR